packages/website/src/docs/box.mdx

```diff
@@ -1,18 +1,21 @@
 ---
 title: Box
 component: Box
+description: A primitive useful for layout.
```

Comment:
`Useful for layout` (seems to be the format used for most of the description, let's be consistent)

Reply:
But not everything is a "primitive", like this one. Don't you think it's worth highlighting it?

Reply:
I don't feel like it makes a difference for whoever is using these components no.